includes/VectorTemplate.php

```diff
@@ -162,13 +164,23 @@ public function execute() {
 		<div id="mw-navigation">
 			<h2><?php $this->msg( 'navigation-heading' ) ?></h2>
 
-			<div id="mw-head" style="top:35px;">
+			<div id="mw-head">
 				<style>
 					#app-brand-name:before{
-						content: ''; display: block; position: absolute; left: 166px; top: 0; width: 0; height: 0; border-top: 20px solid transparent; border-bottom: 22px solid transparent; border-left: 20px solid #6D4E86; clear: both;
+						content: ''; display: block; position: absolute; left: 197px; top: 0; width: 0; height: 0; border-top: 20px solid transparent; border-bottom: 22px solid transparent; border-left: 20px solid #6D4E86; clear: both;
```

Comment:
this this appears about 8px apart in firefox vs. chrome

Reply:
Can you send me a screenshot of what you are seeing? I am not seeing it on my end

Reply:
looks like I'm getting that because the same font size (24px in this case) is for some reason about 10% larger than in chromium for me, couldn't find any settings / mods that might have done it though.